modules/monitoring-creating-scrape-sample-alerts.adoc

```diff
@@ -0,0 +1,75 @@
+:_module-type: PROCEDURE
+// Module included in the following assemblies:
+//
+// * assemblies/configuring-the-monitoring-stack.adoc
+
+[id="creating-scrape-sample-alerts_{context}"]
+= Creating scrape sample alerts
+
+[role="_abstract"]
+You can create alerts that notify you when:
```

Comment:
In OSD/ROSA, using alerts for notification is unavailable. The alerts can show up in console, but there's no way for customers to configure sending the alerts anywhere

Reply:
@fahlmant  Which console please?

Reply:
The Alert tab under the monitoring tab. However, we don't usually recommend customers look there, as they show alerts that are meant for SREP. But if the customer creates an alert, it will show up there (it will not ping SREP)

Reply:
OK, assuming this is the OpenShift console > Alerts tab, not the OCM console. I added it because if there is no other way for them to see or receive and alert, then I;m not sure why we're telling them they can create an aelert. maybe we're better off leaving out this topic altogether?

Reply:
Not the OCM console.

Yes, I agree we should leave it out until alert routing is available.

Reply:
Ok, I deleted that topic for this release. Thanks very much.